### PR TITLE
Getting started refresh

### PR DESCRIPTION
**Pull request by maasg**
This PR refreshes the getting started guide to use the new features in Cloudflow 2.0

### CODE REVIEW

**Comment by maasg** [comment]
This is looking much better now.
image

File: docs/docs-source/docs/modules/get-started/pages/create-example-blueprint.adoc [hunk 1]
```diff
@@ -10,24 +10,39 @@ ifdef::review[REVIEWERS: it would be great to provide some comments explaining t
...
+----
+include::{cloudflow-examples-version}@docsnippets:ROOT:example$sensor-data-scala/src/main/blueprint/blueprint.conf[tag=all]
+----
+
+
+== Understanding the Blueprint File.
+The blueprint file contains two sections: the `streamlets` section and the `topics` section.
+
+=== The `streamlets` section
+In the `streamlets` section of the blueprint, we declare the streamlets participating in the pipeline definition.
```
Comment (RayRoestenburg) [inline docs/docs-source/docs/modules/get-started/pages/create-example-blueprint.adoc:23]: Can we use `application` instead of `pipeline definition`?

Comment (maasg) [reply docs/docs-source/docs/modules/get-started/pages/create-example-blueprint.adoc:23]: we could. But "application" is everywhere. This is a bit more specific.

File: docs/docs-source/docs/modules/get-started/pages/create-example-blueprint.adoc [hunk 2]
```diff
@@ -10,24 +10,39 @@ ifdef::review[REVIEWERS: it would be great to provide some comments explaining t
...
+
+=== The `streamlets` section
+In the `streamlets` section of the blueprint, we declare the streamlets participating in the pipeline definition. 
+Each streamlet is declared by: `name = <fully.qualified.name.of.the.implementing.class>`
+where the streamlet `name` is a short descriptive name, which must be unique within the pipeline definition.
+
+=== The `topics` section
+
+In the `topics` section of the blueprint, we declare the Kafka topics used in this pipeline and the producers and consumers attached to it.
+Streamlet outlets are producers. Streamlet inlets are consumers. Consumers and producers of the same topic must have the same or compatible schemas.
```
Comment (RayRoestenburg) [inline docs/docs-source/docs/modules/get-started/pages/create-example-blueprint.adoc:30]: must have -> must use?

File: docs/docs-source/docs/modules/get-started/pages/create-example-blueprint.adoc [hunk 3]
```diff
@@ -36,15 +51,44 @@ Next, commit your changes:
...
 . Use the `git commit` command to commit your changes:
 +
 ----
 git commit -m "my first Cloudflow app"
 ----
 
+== Verify the Blueprint (optional)
+
+Cloudflow allows you to verify the sanity of the blueprint before you use the application.
+The verification process checks for unconnected end points, schema compatibility between streamlets, invalid/non-existing streamlets, and other issues related to the structure and semantics of the pipeline.
```
Comment (RayRoestenburg) [inline docs/docs-source/docs/modules/get-started/pages/create-example-blueprint.adoc:65]: would also prefer application over pipeline here.

File: docs/docs-source/docs/modules/get-started/pages/define-avro-schema.adoc [hunk 4]
```diff
@@ -5,125 +5,58 @@
...
 * **SensorData:** The data that we receive from the source and ingest through our ingress.
 * **Measurements:** A substructure of `SensorData` that abstracts the values of interest, such as wind speed.
-* **Metric:** A domain object that identifies the Turbine we would like to track and measure.
-* **InvalidMetric:** An object that abstracts the erroneous metric.
+* **Metric:** A domain object that identifies the reported values we would like to track and measure.
+* **InvalidMetric:** An object that encapsulates the erroneous metric.
 
-During the build process, the Cloudflow plugin system processes the schemas. For each of these schema files, Cloudflow will generate Scala case classes that can be directly used from within the application.
+During the build process, the Cloudflow plugin system processes the schemas. 
+For each of these schema files, Cloudflow will generate Scala case classes that can be directly used from within the application.
```
Comment (RayRoestenburg) [inline docs/docs-source/docs/modules/get-started/pages/define-avro-schema.adoc:19]: Scala or Java classes ?

Comment (maasg) [reply docs/docs-source/docs/modules/get-started/pages/define-avro-schema.adoc:19]: In the context of the getting-started, these are Scala 'case classes'.

File: docs/docs-source/docs/modules/get-started/pages/deploy-to-gke-cluster.adoc [hunk 5]
```diff
@@ -1,68 +1,79 @@
...
 
 == Deploy Application to the Cluster
 
-This is quite straightforward and the output of `buildAndPublish` actually tells you what command to run for deployment. In addition you need to supply the credentials along with the CLI.
+This is quite straightforward and the output of `buildApp` actually tells you what command to run for deployment. 
+In addition you need to supply the credentials to the Docker repository along with the CLI `deploy` command.
 
 ```
-$ kubectl-cloudflow deploy -u oauth2accesstoken eu.gcr.io/<gcloud project id>/sensor-data:2-89ce8a7 -p "$(gcloud auth print-access-token)"
+$ kubectl cloudflow deploy /path/to/sensor-data-scala/target/sensor-data-scala.json -u oauth2accesstoken -p "$(gcloud auth print-access-token)"
```
[the remark this answers is not in the document]
Comment (maasg) [reply docs/docs-source/docs/modules/get-started/pages/deploy-to-gke-cluster.adoc:51]: Looks like it's not even required on GKE. 
If you give `user "", pwd "" `, it also works given that it's all GKE.

File: docs/docs-source/docs/modules/get-started/pages/wind-turbine-example.adoc [hunk 6]
```diff
@@ -5,30 +5,61 @@
...
 
-* Ingestion by an Ingress streamlet: An Ingress streamlet has zero inlets and one or more outlets. An ingress could be a server handling requests e.g. using http. In this example, it ingests the wind turbine events.
-* Conversion by a Processor streamlet into a domain object: A Processor streamlet has one inlet and one outlet. Processors represent common data transformations like map and filter, or any combination of them. In this example, the Processor converts the event data into metrics that can be further validated. 
-* Validation by a FanOut streamlet: A FanOut (also known as a Splitter) has one inlet and more than one outlet. The FanOut splits the input into multiple data streams depending on some criteria. The example Splitter separates valid and invalid records.
-* Logging to Egress streamlet: An Egress represents data leaving the application. For instance this could be data being persisted to some database, notifications being sent to Slack, files being written to HDFS, etc. In this case, the streamlet will log data to be checked at output.
+* Ingestion by an Ingress streamlet: An Ingress streamlet has zero inlets and one or more outlets.
```
[the remark this answers is not in the document]
Comment (maasg) [reply docs/docs-source/docs/modules/get-started/pages/wind-turbine-example.adoc:23]: yep

File: docs/docs-source/docs/modules/get-started/pages/wind-turbine-example.adoc [hunk 7]
```diff
@@ -5,30 +5,61 @@
...
+The development of a Cloudflow application typically starts by roughly defining the different blocks of the streaming pipeline, that we translate into streamlets, and the schema that they use to communicate between them. 
 
-One of the important features of Cloudflow architecture is the complete separation of the components from how they are connected as part of the pipeline. The _streamlets_ described above are the individual building blocks of the pipeline. You can connect them using a declarative language that forms the _blueprint_ of the pipeline. Streamlets can be shared across blueprints making them individual reusable objects. And just like streamlets, blueprints also form an independent component of a Cloudflow application.
+.The conceptual starting point of a Cloudflow application
+image::creating-a-cloudflow-app.jpg[Example application]
 
-This example demonstrates using Akka streamlets implemented in Scala. We'll follow these steps:
+Then, we formalize the schema definitions using AVRO, compile it, and use the generated types to represent the inputs and outputs of the streamlets.
```
[the remark this answers is not in the document]
Comment (maasg) [reply docs/docs-source/docs/modules/get-started/pages/wind-turbine-example.adoc:44]: not in the scope of the getting started :-)

File: docs/docs-source/docs/modules/get-started/pages/deploy-to-gke-cluster.adoc (hunk 5 above)
[the remark this answers is not in the document]
Comment (RayRoestenburg) [reply docs/docs-source/docs/modules/get-started/pages/deploy-to-gke-cluster.adoc:51]: you should be able to leave user and pwd out completely then?

**Pull request merged**